src/main/resources/application.yaml

```diff
@@ -102,6 +102,7 @@ service-config:
         - DIVORCE
         - DIVORCE_NOTICE_OF_ACTING
       transformation-url: ${TRANSFORMATION_URL_DIVORCE}
+      allow-creating-case-before-payments-are-processed: true
```

Comment:
should this be enabled per environment ?

Reply:
I don't think so. For probate, we just have it in applciation.yaml file